pyquil/_parser/grammar.lark

```diff
@@ -70,6 +72,8 @@ def_calibration : "DEFCAL" name params qubit_designators ":" indented_instrs
 def_measure_calibration : "DEFCAL" "MEASURE" qubit_designator [ name ] ":" indented_instrs
 
 gate : [ modifiers ] name [ params ] qubit_designators
+bare_gate.2 : name 
```

Comment:
what's the `.2`?

Reply:
It gives that rule a higher priority. `bare_gate` and `gate` look similar enough up to their `name` items so that lark needs a way to choose which rule to follow if both match up to that part. (I had thought that was decided by the possible lengths of the matches, but apparently not quite... or I'm misunderstanding.)